### ISSUE TITLE
Error with Layout/SpaceInsideArrayLiteralBrackets

### ISSUE THREAD

**Issue by QuentinFchx** [bbatsov/rubocop#5637]
When inspecting code such as:

```ruby
module Foo
    def bar
        ActiveRecord::Base.connection.execute(<<-SQL).hmap { |row| [row['name'], row['count'].to_i] }
            SELECT
                name,
                COUNT(DISTINCT baz.id) AS count
            FROM foo
        SQL
    end
end
```
--------

## Expected behavior

The linter should work

## Actual behavior

`undefined method '+' for nil:NilClass` is thrown.
```
ruby/2.5.0/gems/rubocop-0.53.0/lib/rubocop/cop/layout/space_inside_array_literal_brackets.rb:122:in `next_to_newline?'
ruby/2.5.0/gems/rubocop-0.53.0/lib/rubocop/cop/layout/space_inside_array_literal_brackets.rb:79:in `on_array'
```

## Steps to reproduce the problem

See code snippet above.

## RuboCop version
 
`0.53.0 (using Parser 2.5.0.3, running on ruby 2.5.0 x86_64-darwin17)`
